storage-model/src/main/java/no/unit/nva/publication/storage/model/daos/DoiRequestDao.java

```diff
@@ -25,6 +25,19 @@ public DoiRequestDao(DoiRequest doiRequest) {
         this.data = doiRequest;
     }
 
+    public static DoiRequestDao queryObject(URI publisherId, String owner, SortableIdentifier doiRequestIdentifier) {
+        DoiRequest doi = new DoiRequest(
+            doiRequestIdentifier,
+            doiRequestIdentifier,
```

Comment:
Null Resource Identifier is not allowed. Subsequent PR has a fix on this with a private constructor and more static generator methods